govc/USAGE.md

```diff
@@ -38,6 +38,8 @@ but appear via `govc $cmd -h`:
 
  - [about](#about)
  - [about.cert](#aboutcert)
+ - [alarm.info](#alarminfo)
+ - [alarms](#alarms)
```

Comment:
"alarms" is defined in "alarms/state.go". We have similar with "events" and "tasks", command name is shorter than the package.type name.

"create" and "rm" are registered with the `unreleased` flag, see:
https://github.com/vmware/govmomi/blob/fc7056a74e59b8c57496b0987cc2ddbf6d5d323f/govc/cli/register.go#L25-L27

Chose to keep create unreleased for now since it can only create with EventAlarmExpression, does not handle State or Metric expressions. And didn't want rm to be confused for "reset to green" in the UI. Seems it would be rare for users to remove an alarm def. We can consider releasing those commands in the future, but for now they're mainly useful developers using govmomi.

Reply:
Got it, and thanks for the clarification.